AndroidSDKTests/src/test/java/com/leanplum/LeanplumTest.java

```diff
@@ -1455,4 +1457,21 @@ public void testParseLastStartResponseGivenMultipleStartShouldReturnResponse() t
     }
     return responsesList;
   }
+
+  /**
+   * Test trigger message displayed calls callback
+   */
+  @Test
+  public void testTriggerMessageDisplayed() {
```

Comment:
This tests a callback, not display, right? Misnomer? Also can we test that the callback has all the information from the business requirements for archiving?

Reply:
To keep the method name short, I put the specific information in the comment above. `triggerMessageDisplayed` is the specific method being tested here. I can make the method name more descriptive again.